lib/teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I don't think it is necessary to use `associations`(**"attr_accessor :specialization"**) in this file, so it would be better if you remove them too 😃

Reply:
@mukhammadrizooff I was actually thinking that later on maybe I need to use `getter` and `setter`.

Reply:
@mukhammadrizooff I fixed it Thanks ❤️